usaspending_api/api_docs/markdown/using_the_api.md

```diff
@@ -50,6 +50,11 @@ The currently available endpoints are listed below. To reduce unnecessary data t
     - _Methods_: GET, POST
 
 
+  * **/v1/accounts/tas/autocomplete/**
```

Comment:
Thanks for updating the docs. Do you think this (or one of the other autocomplete endpoints) is worthy of a cookbook entry?